openupgrade_scripts/scripts/base/14.0.1.3/pre-migrate.py

```diff
@@ -123,6 +137,7 @@ def migrate(cr, version):
     """
     )
     # Perform module renames and merges
+    uninstall_conflicting_it_edi(cr)
     openupgrade.update_module_names(cr, renamed_modules.items())
     openupgrade.update_module_names(cr, merged_modules.items(), merge_modules=True)
     # Migrate partners from Fil to Tagalog
```

Comment:
I think it should go here the uninstall_conflicting_it_edi(cr) call instead.

Reply:
Thanks for the review!
The modules are not renamed so I didn't think of that, but I'm talking about `14.0` modules so it makes sense to move after the renaming.
I have moved it, please check if it's better now :)